### PR TITLE
fix a bunch of structure code error

### PR DESCRIPTION
**Pull request by Glease**
probably not the last of this series...

### CODE REVIEW

File: src/main/java/gtPlusPlus/xmod/gregtech/common/tileentities/machines/multi/production/GregtechMetaTileEntity_AlloyBlastSmelter.java [hunk 1]
```diff
@@ -117,7 +117,7 @@ public IStructureDefinition<GregtechMetaTileEntity_AlloyBlastSmelter> getStructu
 					.addElement(
 							'C',
 							buildHatchAdder(GregtechMetaTileEntity_AlloyBlastSmelter.class)
-									.atLeast(InputBus, OutputBus, Maintenance, Energy, Muffler)
+									.atLeast(InputBus, InputHatch, OutputBus, Maintenance, Energy, Muffler)
```
Comment (chochem) [inline src/main/java/gtPlusPlus/xmod/gregtech/common/tileentities/machines/multi/production/GregtechMetaTileEntity_AlloyBlastSmelter.java]: needs Output hatch too

Comment (Glease) [reply src/main/java/gtPlusPlus/xmod/gregtech/common/tileentities/machines/multi/production/GregtechMetaTileEntity_AlloyBlastSmelter.java]: Actually, isn't this fluid output only?

Comment (chochem) [reply src/main/java/gtPlusPlus/xmod/gregtech/common/tileentities/machines/multi/production/GregtechMetaTileEntity_AlloyBlastSmelter.java]: needs both, for example for rhenium

File: src/main/java/gtPlusPlus/xmod/gregtech/common/tileentities/machines/multi/processing/GregtechMetaTileEntity_IndustrialArcFurnace.java [hunk 2]
```diff
@@ -106,7 +106,7 @@ public IStructureDefinition<GregtechMetaTileEntity_IndustrialArcFurnace> getStru
 					.addElement(
 							'C',
 							buildHatchAdder(GregtechMetaTileEntity_IndustrialArcFurnace.class)
-									.atLeast(InputBus, OutputBus, Maintenance, Energy, Muffler)
+									.atLeast(InputBus, InputHatch, OutputBus, Maintenance, Energy, Muffler)
```
Comment (chochem) [inline src/main/java/gtPlusPlus/xmod/gregtech/common/tileentities/machines/multi/processing/GregtechMetaTileEntity_IndustrialArcFurnace.java]: I think this also needs an output hatch when used as plasma arc furnace

**Review by chochem — APPROVED** [review]
have not looked at all of them, but volcanus, cutting machine, arc furnace, and ABS are fixed

**Pull request merged**